secretstore/secretstore.go

```diff
@@ -129,3 +129,22 @@ func SecretFromBytes(b []byte) (*Secret, error) {
 
 	return &Secret{s: s}, nil
 }
+
+// Plaintext decrypts and returns the secret value as a byte slice for
+// the given secret store and secret name.
+//
+// This is a convenience function that combines the functionality of
+// [Open], [Store.Get], and [Secret.Plaintext].
```

Comment:
Do we want to add a caveat if that if you're obtaining multiple secrets from a store to actually call `Open/Get/Plaintext` etc instead of calling this repeatedly?

Reply:
IMO we can skip it, but I'll add it if you feel strongly about it.  There's very little overhead, basically just the `open` hostcall itself and copying a handle or error back into guest space.

Reply:
Also, 🚲 🏠 ⚠️ but I prefer phrasing it as `returns the secret value as a []byte for`